src/nvim/os/unix_defs.h

```diff
@@ -28,46 +32,51 @@
 #ifndef SYNTAX_FNAME
 # define SYNTAX_FNAME   "$VIMRUNTIME/syntax/%s.vim"
 #endif
+// Not used anymore ?
```

Comment:
`USR_EXRC_FILE`, `USR_VIMRC_FILE2`, `EXRC_FILE`,  can be removed. But not `VIMRC_FILE`.